Snowdog_Components/components/Molecules/table/_table.scss

```diff
@@ -84,6 +84,25 @@
         }
     }
 
+    &--tfoot-border {
+        tfoot {
+            tr {
+                th {
+                    border-top: $table__tfoot-th-border-top\@medium;
```

Comment:
`\@medium` means that the style is applied for `@include mq($screen-m)` but I can't see any media queries here so it should be removed. one thing worth checking first is whether the styles look good on mobile. can you confirm if there aren't any doubled borders?